dss-api.yml

```diff
@@ -996,23 +996,75 @@ paths:
                 required:
                   - code
   /collections:
+    get:
+      operationId: dss.api.collections.listc
```

Comment:
is this supposed to be `dss.api.collections.listcollection` ?

Reply:
No, but I can change it!

Reply:
Addressed.